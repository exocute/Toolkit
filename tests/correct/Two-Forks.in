Graph Two_Forks
Import Int
Export Double
	Activity A com.classes.A:10
	Import Int
	Export String
	
	Activity B com.classes.B
	Import String
	Export Int
		Activity C com.classes.C:60:1:134
		Import Int
		Export String
		Activity D com.classes.D
		Import Int
		Export Int
	Activity E com.classes.E:param1:param2:param3
	Import String
	Export String
	
	Activity F com.classes.F
	Import String, Int, String
	Export Double
	
Connection A -> B -> (C,D) -> F
Connection A -> E -> F