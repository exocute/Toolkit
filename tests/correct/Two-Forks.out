Graph Two_Forks
Import Int
Export Double
Activity A com.classes.A:10
Import Int
Export String, Float
Activity B com.classes.B
Import String
Export Int, Float
Activity C com.classes.C:60:1:134
Import Int
Export String
Activity D com.classes.D
Import Float
Export Int
Activity E com.classes.E:param1:param2:param3
Import Float
Export String
Activity F com.classes.F
Import String, Int, String
Export Double
Connection A -> B, E
Connection B -> C, D
Connection C -> F
Connection D -> F
Connection E -> F